### ISSUE TITLE
Add info about .citation.cff files on github

### ISSUE THREAD

**Issue by lwasser** [pyOpenSci/python-package-guide#34]
add this to the license page:

- [ ] how .cff files work - what they add to the repo
- [ ] how dates are tracked in this file
- [ ] how they integrate into / with zenodo

**Comment by SanketKumarKar** [comment]
What is a CITATION.cff file?
A CITATION.cff file is a simple, human- and machine-readable file (written in YAML format) that you add to the root of your repository. Its main purpose is to tell others how to cite your software correctly. When you include a CITATION.cff file, GitHub will automatically show a “Cite this repository” button, making it easy for users to find the citation information.

What does a CITATION.cff file add to your repository?
Citation Instructions: It provides clear instructions on how to cite your work.
Metadata: It contains important metadata about your project, such as authors, title, version, and more.
Machine Readability: Citation tools and services (like GitHub and Zenodo) can read this file to generate standardized citation formats automatically.
How are dates tracked in CITATION.cff?
The CITATION.cff file can include fields such as date-released (and optionally, date-created and date-modified). These dates help others know when your code was published, and they are included in citations generated from the file.

Example:

YAML
date-released: "2024-06-01"
How does CITATION.cff integrate with Zenodo?
Zenodo is a popular service for archiving research outputs and assigning DOIs (Digital Object Identifiers). When you connect your GitHub repository to Zenodo, Zenodo reads the CITATION.cff file (if present) and uses the metadata to create accurate citations for each version it archives. This ensures consistency and saves you time, as you only need to maintain your citation info in one place.

In summary:
Adding a CITATION.cff file to your repository helps others cite your work properly, tracks important dates, and ensures your citations are up-to-date and consistent across platforms like GitHub and Zenodo.

**Comment by lwasser** [comment]
@all-contributors please add @SanketKumarKar for code

**Comment by lwasser** [comment]
hey there @SanketKumarKar this looks like a great start to this issue. Are you open to submitting a pull request for it? We can then have folks in our community review.

I also don't recall if you wanted to be invited to our slack community. If you do, I can invite you (i just need an email) and we can discuss the pr further there if you'd like some support in submitting it. Please let me know!

**Comment by lwasser** [comment]
The next steps in this issue is to evaluate / review the text above. We likely want to edit/enhance it to match the tone of our guide. Then we can add it to the guide as a formal pull request. 

I think we also want to add some context around the file and link to an example of it.

**Comment by schwalbe10** [comment]
I'll try it today!

**Comment by SanketKumarKar** [comment]
Hello, I am ready to create pull requests

**Comment by schwalbe10** [comment]
Oh, I see.

**Comment by lwasser** [comment]
hey @SanketKumarKar we are sprintig today and @schwalbe10 is goig to take the text that you provided and update it some and then submit a PR. Can you please review the PR after it's submitted? we are working together in real time today at a conference.

**Comment by SanketKumarKar** [comment]
> hey [@SanketKumarKar](https://github.com/SanketKumarKar) we are sprintig today and [@schwalbe10](https://github.com/schwalbe10) is goig to take the text that you provided and update it some and then submit a PR. Can you please review the PR after it's submitted? we are working together in real time today at a conference.



Okiee I will review the PR

**Comment by lwasser** [comment]
Oh this got closed last week. @malvikasharan i will turn your comment into a new issue
